Lib/dis.py

```diff
@@ -1 +1,12 @@
-from _dis import *
+import argparse
+import _dis
+
+
+# Disassembling a file by following cpython Lib/dis.py
+parser = argparse.ArgumentParser()
+parser.add_argument('infile', type=argparse.FileType('rb'), nargs='?', default='-')
+args = parser.parse_args()
+with args.infile as infile:
+    source = infile.read()
+code = compile(source, args.infile.name, "exec")
+_dis.dis(code)
```

Comment:
This file is not only used for the command `rustpyhon -m dis` but also used as an independent module like `import dis` in common python code.
So this new codes must be isolated by guarding `if __name__ == "__main__":` just like CPython handle it.